sakai-evaluation-tool/src/java/org/sakaiproject/evaluation/tool/renderers/NavBarRenderer.java

```diff
@@ -81,7 +97,19 @@ public void makeNavBar(UIContainer tofill, String divID, String currentViewID) {
         if (isUserAdmin) {
             renderLink(joint, AdministrateProducer.VIEW_ID, "administrate.page.title");
         }
-        
+
+        // Provide logout and my workspace links
+        if( !isUserAdmin && !adminAllowedToSee && (!canCreateTemplate && !canBeginEvaluation) )
```

Comment:
(!canCreateTemplate && !canBeginEvaluation) - brackets are unnecessary; whole expression only evaluates to true if all items are false